driver/compmisc.ml

```diff
@@ -41,31 +44,17 @@ let init_path ?(auto_include=auto_include) ?(dir="") () =
        Config.flexdll_dirs;
        !Compenv.first_include_dirs]
   in
-  let visible =
-    List.map (Misc.expand_directory Config.standard_library) visible
-  in
   let visible =
     (if !Clflags.no_cwd then [] else [dir])
     @ List.rev_append visible (Clflags.std_include_dir ())
   in
-<<<<<<< HEAD
   let hidden =
     List.rev_map (Misc.expand_directory Config.standard_library)
       !Clflags.hidden_include_dirs
+    @ List.rev_append visible (Clflags.std_include_dir ())
```

Comment:
I don't quite understand what happened in the merge resolution here - I think you want to keep the deleted lines 44-46 and then take the HEAD version here unchanged.